cmd/neofs-node/config/engine/config.go

```diff
@@ -58,3 +58,10 @@ func ShardPoolSize(c *config.Config) uint32 {
 
 	return ShardPoolSizeDefault
 }
+
+// ShardErrorThreshold returns value of "shard_ro_error_threshold" config parameter from "storage" section.
+//
+// Returns 0 if the value is missing.
```

Comment:
I think it is not good to be fault intolerant by default, but I don't have any good default value for that, 10, 100, who knows...

Reply:
I think this non-zero default is good, but it should be provided in config, shard should not define any policy by itself IMO.

Reply:
`neofs-node/config` package was designed for default app configuration, so it fits there. But still don't know what would be a good value for that.